index.d.ts

```diff
@@ -916,6 +916,11 @@ declare namespace Eris {
     (event: "resume", listener: () => void): T;
   }
 
+  interface ChannelFollow {
+    channel_id: string;
```

Comment:
This might want to be looked at if we provide snake case anywhere else. I don't think Eris does and we should try to be consistant and modify this before getting it to the end user.

Reply:
Irrelevant, this is what Discord API returns

Reply:
Though.. thinking about it I'll ask abal later. I don't want to alter what the API returns too much

Reply:
bbb

Reply:
Actually, after having a look at it, snake_case is used, see FriendSuggestionReasons, Activity, OAuthApplicationInfo etc